foia_hub/templates/about.html

```diff
@@ -61,10 +61,9 @@
 
             <h2>Feedback</h2>
 
-            <p>We welcome and encourage feedback! Please reach out to us: <a
-            href="mailto:18f-foia@gsa.gov">18f-foia@gsa.gov</a></p>
-
-            <hr />
+            <p>We welcome and encourage feedback! Please reach out to us
+            via email <a href="mailto:18f-foia@gsa.gov">18f-foia@gsa.gov</a> or on our Github repository <a href="https://github.com/18F/foia-hub/issues"> <span class="fa fa-github"></span> foia-hub</a>
```

Comment:
I would recommend  changing this to 'or through our issue tracker' and change the link to say foia-hub/issues